openml/tasks/functions.py

```diff
@@ -329,25 +335,30 @@ def get_task(task_id: int, download_data: bool = True) -> OpenMLTask:
 
     Parameters
     ----------
-    task_id : int or str
-        The OpenML task id.
-    download_data : bool
+    task_id : int
+        The OpenML task id of the task to download.
+    download_data : bool (default=True)
         Option to trigger download of data along with the meta data.
+    download_qualities : bool (default=True)
+        Option to download 'qualities' meta-data in addition to the minimal dataset description.
 
     Returns
     -------
     task
     """
+    if not isinstance(task_id, int):
+        warnings.warn("Task id must be specified as `int` from 0.14.0 onwards.", DeprecationWarning)
```

Comment:
Went with (what I remember to be) scikit-learn's default behavior of having at least one full minor release with the deprecation warning.